.github/workflows/ci.yml

```diff
@@ -12,7 +12,7 @@ jobs:
         rust:
           - stable
           - beta
-          - 1.66.1 # MSRV
+          - 1.67.1 # MSRV
```

Comment:
1.70.0

Reply:
cargo msrv yielded `1.67.1`, so i fixed the `rust-toolchain` to match it as well